### ISSUE TITLE
Failing to finetune

### ISSUE THREAD

**Issue by devloper13** [WongKinYiu/yolor#49]
Here's what I did:
1. I have a pascal voc format label (8 classes, completely different form coco labels), which I converted to yolo format using this: https://gist.github.com/Amir22010/a99f18ca19112bc7db0872a36a03a1ec

2. I made changes to yolor_p6.cfg, replacing 255 with appropriate value, in my case 39, and the num_classes.
3. I replaced labels in coco.labels with 8 classes of my own.
4. The train command I used:
`python train.py --batch-size 8 --img 1280 1280 --data coco.yaml --cfg cfg/yolor_p6.cfg --weights '' --device 0 --name yolor_p6 --hyp hyp.scratch.1280.yaml --epochs 300`

Also tried with hyp.finetune.1280.yaml

But the model doesn't learn anything. Infact after 12 epochs the losses start increasing and mAP starts falling, which was already bad to begin with. Can you please guide me?
